examples/02_data/01_read_dataset.py

```diff
@@ -33,14 +31,17 @@
 
 def main(backend: str) -> None:
     """Read intermediate file using `Dataset` class."""
+    # Construct Logger
+    logger = Logger()
+
     # Check(s)
     assert backend in DATASET_CLASS
 
     path = TEST_SQLITE_DATA if backend == "sqlite" else TEST_PARQUET_DATA
     pulsemap = "SRTInIcePulses"
     truth_table = "truth"
-    batch_size = 128
-    num_workers = 30  # 30
+    batch_size = 1  # 128
```

Comment:
`batch_size` variable should be reverted from testing parameters before merging.

Reply:
What an excellent find, @MortenHolmRep. Thanks! It's fixed now.